config.default.yaml

```diff
@@ -22,6 +22,8 @@ enable:
   # requires cds API key https://cds.climate.copernicus.eu/api-how-to
   # More information https://atlite.readthedocs.io/en/latest/introduction.html#datasets
 
+custom_rules: []
```

Comment:
Maybe worth adding a comment here to describe what it may contain and that its default value is an empty list.
Similarly below to the tutorial file.

Reply:
done